webapp/graphite/storage.py

```diff
@@ -33,8 +34,9 @@ def __init__(self, finders=None, hosts=None):
     self.remote_stores = [ RemoteStore(host) for host in remote_hosts ]
 
 
-  def find(self, pattern, startTime=None, endTime=None, local=False):
+  def find(self, pattern, startTime=None, endTime=None, local=False, user=None):
```

Comment:
When `user` isn't provided, `user=None` breaks the `nodeFilter` function. I think you probably want to make `user` a required argument and update all `STORE.find()` occurrences to provide `user`.